app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/CustomSelectorActivity.kt

```diff
@@ -67,6 +71,22 @@ class CustomSelectorActivity : BaseActivity(), FolderClickListener, ImageSelectL
      */
     private lateinit var prefs: SharedPreferences
 
+    /**
+     * Maximum number of images that can be selected.
+     */
+    private val imageUploadLimit: Int = 20
+
+    /**
+     * Flag that is marked true when the amount
+     * of selected images is greater than the upload limit.
+     */
+    private var overUploadLimit: Boolean = false
```

Comment:
One more thing, how about renaming these 2 variables to uploadLimit, uploadLimitExceeded?
It would help the readers understand effortlessly that the 3 variables are related to the same feature.
Thanks a lot!